src/commands/migrateContent/run/run.ts

```diff
@@ -143,7 +145,19 @@ type MigrateContentRunCliParams =
 const migrateContentRunCli = async (params: MigrateContentRunCliParams) => {
   const resolvedParams = resolveParams(params);
 
-  migrateContentRunIntenal(resolvedParams, "migrate-content-run");
+  migrateContentRunInternal(resolvedParams, "migrate-content-run", async () => {
+    const warningMessage = chalk.yellow(
+      `⚠ Running this operation may result in irreversible changes to the content in environment ${params.targetEnvironmentId}. 
+OK to proceed y/n? (suppress this message with --sw parameter)\n`,
+    );
+
+    const confirmed = !params.skipConfirmation ? await requestConfirmation(warningMessage) : true;
+
+    if (!confirmed) {
+      logInfo(params, "standard", chalk.red("Operation aborted."));
+      process.exit(1);
```

Comment:
This pattern of `requestConfirmation` => abort if false is pretty common already. What about abstracting it into a function? e.g. `checkConfirmation`